tests/integration/protostar_fixture.py

```diff
@@ -323,6 +340,28 @@ def create_files(
                 content = file
             self._save_file(self._project_root_path / relative_path_str, content)
 
+    def set_cached_configuration_file(
```

Comment:
Do we use it anywhere outside of tests?

Reply:
It's a `ProtostarFixture`. Why should it be used outside test context?